lib/money-rails/helpers/action_view_extension.rb

```diff
@@ -48,5 +48,16 @@ def money_without_cents(value, options={})
     def money_without_cents_and_with_symbol(value)
       money_without_cents(value, symbol: true)
     end
+
+    def money_only_cents(value)
+      if value.is_a?(Money)
```

Comment:
There's probably no need for this empty branch

Reply:
thank you, i didn't realize that `money.to_money == money`.